src/xrCore/file_stream_reader.cpp

```diff
@@ -37,8 +37,10 @@ void CFileStreamReader::destroy()
     CloseHandle(file_mapping_handle);
     CloseHandle(m_file_handle);
 #elif defined(XR_PLATFORM_LINUX)
-    int file_mapping_handle = this->file_mapping_handle();
```

Comment:
Behaviour should not be changed in a "polishing" PR, unless it's clearly fixes a warning that indicates a real bug.
Did the compiler thew warnings here?

I think, it's better not to change the code here, but we can leave `XXX` to look at it later or in another PR :)

Reply:
@Masterkatze, but what about leaving the comment, like: `XXX: maybe, we should close the handle` ?

Reply:
@Xottab-DUTY we open file_mapping_handle only on Windows (line 15), so everything is in place now, no changes needed.